src/Form/Form.tsx

```diff
@@ -1,9 +1,56 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./Form.css";
 
-const Form = () => {
+export interface Quote {
+  q: string;
+  a: string;
+  h: string;
+}
+
+interface formProps {
+  setAllQuotes: React.Dispatch<React.SetStateAction<Quote[]>>
+}
+
+
+interface Mood{
+  k: string;
+  l: string;
+}
+
+const Form = ({setAllQuotes}: formProps) => {
+  const [moods, setMoods] = useState<Mood[]>([])
+
+  useEffect(() => {
```

Comment:
Nice work! Cool use of useEffect to set the moods with the keywords from the API